src/layouts/ScaleStrategies.js

```diff
@@ -81,8 +82,18 @@ define(function() {
                 }
             }
 
+            var marginTop;
```

Comment:
Edge case, but you could probably just return 0 here if maxPageHeight is 0 to avoid division by 0 on line 95. This would only really happen when there are no items.